piccolo/apps/migrations/auto/diffable_table.py

```diff
@@ -188,7 +189,7 @@ def __sub__(self, value: DiffableTable) -> TableDelta:
                         tablename=self.tablename,
                         column_name=column._meta.name,
                         db_column_name=column._meta.db_column_name,
-                        params=deserialise_params(delta),
+                        params=params,
```

Comment:
I don't fully understand the change here - why doesn't `deserialise_params` work?

Reply:
OK - thanks. I remember the problem with auto migrations not importing the array's base column. I need to figure out what's going on there.